src/index.js

```diff
@@ -74,14 +74,22 @@ module.exports = async function App(config) {
     )}`,
   );
 
-  await sendDelayedMessages(blocks => releaseCommunication.sendMessage(blocks), preparedBlocks);
+  try {
+    await sendDelayedMessages(blocks => releaseCommunication.sendMessage(blocks), preparedBlocks);
+  } catch (error) {
+    console.log(`Failed to send message to release channel ${error}`);
+    throw error;
+  }
 
   // Send all individual attachments to their respective channels per team.
   if (subChannelBlocks.length) {
-    await sendDelayedMessages(
-      ({ channel: subChannel, blocks }) => releaseCommunication.sendMessage(blocks, subChannel, true),
-      subChannelBlocks,
-    );
+    try {
+      await sendDelayedMessages(({ channel: subChannel, blocks }) => {
+        releaseCommunication.sendMessage(blocks, subChannel, true);
+      }, subChannelBlocks);
+    } catch (error) {
+      console.log(`Unable to post to subchannels: ${error}`);
```

Comment:
ditto https://github.com/target/captains-log/pull/99/files#r588806463

```suggestion
      console.error('Unable to post to subchannels', error);
```